### DISCUSSION TITLE
Discord?

### DISCUSSION THREAD

**Discussion by ghost** [DanielChappuis/reactphysics3d#246] [General]
Wondering if there was a discord channel for RP3D.

**Comment by DanielChappuis** [comment]
Hello. No there is no discord channel yet for ReactPhysics3D. You can use the Github Discussions to ask/answer questions.

Do you think it would be useful to have a Discord channel? Why would it be better than Github Discussions? (I don't have a lot of experience with Discord).

**Comment by ghost** [comment]
Hiya.

It can be nice to connect with other people using software, and get casual advice without having to open a full thread.

It's by no means necessary though.

**Comment by MyelinsheathXD** [comment]
Yeah, I think, Discord is cool for chill out with devs. and have a everyday experience with everyone
